logger.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,
+    filename='phonebook.log',
+    format='[%(asctime)s] [%(levelname)s] [%(module)s] [%(funcName)s: %(lineno)d] => %(message)s',
+    datefmt='%d.%m.%Y %H:%M:%S ',
+)
```

Comment:
Не забывайте указывать кодировку.
```suggestion
    datefmt='%d.%m.%Y %H:%M:%S ',
    encoding='utf-8'
)
```